### ISSUE TITLE
[suggestion] chunk for gems

### ISSUE THREAD

**Issue by Biviho** [RCXcrafter/AOBDorechunks#17]
Hi! i know the mod is intended for apply fortune to metals, but it would be cool to also have an option to apply it for gems too, in case a modpack creator want to add a gem cutting mechanic, while not have to sacrifice fortune.

Otherwise, the only other way i have found is to register gems as an ingot with content tweaker, then jaopca and crafttweaker to remove the added recipe for being an ingot, or add a custom chunk item and tweak everything manually

**Comment by RCXcrafter** [comment]
This should be pretty easy to add but you would probably be the only one to use it so I'm not sure if I'll add this

**Comment by Biviho** [comment]
yeah, i supposed it, it is indeed a niece feature, ty for taking the time to respond
